python/tests/test.py

```diff
@@ -308,11 +311,9 @@ def test_get_reports_from_population(self):
         self.assertEqual(keys, [(13, 30), (13, 30), (13, 31), (13, 31), (13, 32), (14, 32), (14, 33), (14, 33), (14, 34), (14, 34)])
 
         self.assertEqual(len(sel.times), 3)  # Number of timestamp (0.8, 1.0 and 1.2)
-        sel = self.test_obj['All'].get(tstart=5., tstop=-1)  # tstart out of range
-        self.assertEqual(len(sel.ids), 0)
-        sel = self.test_obj['All'].get(tstart=3., tstop=3.)
-        self.assertEqual(len(sel.ids), 100)
-
```

Comment:
what happens for `.get(node_ids=[])` ?

Reply:
Actually the behavior is correct imao. This is just about a test lacking.

Reply:
Today, it takes all nodes. Do you prefer an empty `DataFrame`?

Reply:
> Today, it takes all nodes.
ah my bad ! I forgot I hacked bluepysnap for this ... 
Yes having an empty dataframe is what I want. We already have the None to return all nodes then we need something for the empty dataframe and [] is perfect.

Reply:
hm ok